coral-component-multifield/src/templates/item.html

```diff
@@ -1,2 +1,4 @@
 <button id="{{data.uid}}-remove" aria-labelledby="{{data.uid}}-remove {{data.uid}}-content {{data.uid}}" aria-label="{{data.i18n.get('Remove')}}" title="{{data.i18n.get('Remove')}}" tracking="off" type="button" is="coral-button" handle="remove" variant="quietaction" icon="delete" iconsize="S" iconautoarialabel="on" class="_coral-Multifield-remove" coral-multifield-remove></button>
-<button id="{{data.uid}}-reorder" aria-labelledby="{{data.uid}}-reorder {{data.uid}}-content {{data.uid}}" aria-label="{{data.i18n.get('Reorder')}}" aria-roledescription="{{data.i18n.get('reorder_drag_handle')}}" aria-grabbed="false" aria-pressed="false" title="{{data.i18n.get('reorder_hint')}}" tracking="off" type="button" is="coral-button" handle="move" variant="quietaction" icon="moveUpDown" iconsize="S" iconautoarialabel="on" class="_coral-Multifield-move u-coral-openHand" coral-multifield-move></button>
\ No newline at end of file
+<button id="{{data.uid}}-reorder" aria-labelledby="{{data.uid}}-reorder {{data.uid}}-content {{data.uid}}" aria-label="{{data.i18n.get('Reorder')}}" aria-roledescription="{{data.i18n.get('reorder_drag_handle')}}" aria-grabbed="false" aria-pressed="false" title="{{data.i18n.get('reorder_hint')}}" tracking="off" type="button" is="coral-button" handle="move" variant="quietaction" icon="moveUpDown" iconsize="S" iconautoarialabel="on" class="_coral-Multifield-move u-coral-openHand" coral-multifield-move></button>
+<button id="{{data.uid}}-up" aria-labelledby="{{data.uid}}-up {{data.uid}}-content {{data.uid}}" aria-label="{{data.i18n.get('Up')}}" aria-roledescription="{{data.i18n.get('up_drag_handle')}}" aria-grabbed="false" aria-pressed="false" title="{{data.i18n.get('reorder_hint')}}" tracking="off" type="button" is="coral-button" handle="reorderup" variant="quietaction" icon="ChevronUp" iconsize="S" iconautoarialabel="on" class="_coral-Multifield-up u-coral-openHand" coral-multifield-up></button>
+<button id="{{data.uid}}-down" aria-labelledby="{{data.uid}}-down {{data.uid}}-content {{data.uid}}" aria-label="{{data.i18n.get('Down')}}" aria-roledescription="{{data.i18n.get('down_drag_handle')}}" aria-grabbed="false" aria-pressed="false" title="{{data.i18n.get('reorder_hint')}}" tracking="off" type="button" is="coral-button" handle="reorderdown" variant="quietaction" icon="ChevronDown" iconsize="S" iconautoarialabel="on" class="_coral-Multifield-down u-coral-openHand" coral-multifield-down></button>
```

Comment:
I don't think we need a localized `aria-roledescription` for the up / down buttons unless drag and drop still works. Without drag and drop, these are just buttons. Also, the `aria-label` "Up" or "Down" should probably be more descriptive. "Reorder up" or "Reorder down". The localized strings for these should be added to the translations file for this component at https://github.com/adobe/coral-spectrum/blob/master/coral-component-multifield/i18n/translations.js. 

It should also be noted that without independent up or down buttons, the drag and drop interaction is keyboard accessible.  One can focus the drag handle, press <key>Enter</key> or <key>Space</key> to toggle dragging, use <key>ArrowUp</key> or <key>ArrowDown</key> to reorder, and then  <key>Enter</key> or <key>Space</key> stop dragging. However, for mobile screen reader users, on say an iOS device, having independent controls for up or down would be easier than drag and drop.